addons/sale_management/views/sale_order_template_views.xml

```diff
@@ -31,22 +31,27 @@
                     <group>
                         <group name="sale_info">
                             <field name="active" invisible="1"/>
-                            <label for="number_of_days" string="Quotation expires after"/>
+                            <label for="number_of_days" string="Quotation Validity"/>
                             <div id="number_of_days">
                                 <field name="number_of_days" style="width: 3rem;"/>
                                 <div class="d-inline-block">days</div>
                             </div>
-                            <field name="require_signature"/>
-                            <field name="require_payment"/>
-                            <field name="prepayment_percent"
-                                   invisible="not require_payment"
-                                   widget="percentage"
-                                   string="Prepayment amount"
-                                   style="width: 3rem"/>
                             <field name="mail_template_id" context="{'default_model': 'sale.order'}"/>
                             <field name="company_id" options="{'no_create': True}" groups="base.group_multi_company"/>
                             <field name="journal_id"/>
                         </group>
+                        <group name="sale_info2">
```

Comment:
```suggestion
                        <group name="so_confirmation">
```
I believe a clear name w.r.t the group content is better than a vague "sale_info2"